src/main/resources/i18n/Parsing_de.properties

```diff
@@ -0,0 +1,29 @@
+#
+# This resource bundle is used for the query parsing code to produce i18n messages
+#
+# The keys have the format of rule class name and then message type within that.  Most rules
+# will only have 1 or 2 message keys
+#
+# Please try and keep this sorted within rule class and use # between sections so the IDEA Ctrl-Alt-L reformat does not bunch
+# them too tightly.
+#
+# REMEMBER - a single quote ' in MessageFormat means things that are never replaced within them
+# so use 2 '' characters to make it one ' on output.  This will take for the form ''{0}''
+#
+# Prior to Java 9, properties files are encoded in ISO-8859-1.
+# We have to use \u00fc instead of the German ue character, \u00e4 for ae, \u00f6 for oe, \u00df for ss
+#
+InvalidSyntax.noMessage=Ung\u00fcltige Syntax in Zeile {0} Spalte {1}
+InvalidSyntax.full=Ung\u00fcltige Syntax mit ANTLR-Fehler ''{0}'' in Zeile {1} Spalte {2}
```

Comment:
"Mit" sounds a bit strange, but not totally wrong. Maybe just a comma after "Syntax"?